tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Testing/XUnit/V3/XunitTestMethodRunnerContextCtorV3Integration.cs

```diff
@@ -0,0 +1,63 @@
+// <copyright file="XunitTestMethodRunnerContextCtorV3Integration.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+#nullable enable
+using System;
+using System.ComponentModel;
+using System.Threading;
+using Datadog.Trace.Ci;
+using Datadog.Trace.ClrProfiler.CallTarget;
+using Datadog.Trace.DuckTyping;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.Testing.XUnit.V3;
+
+/// <summary>
+/// Xunit.v3.TestCaseRunner`3.RunTest calltarget instrumentation
+/// </summary>
+[InstrumentMethod(
+    AssemblyName = "xunit.v3.core",
+    TypeName = "Xunit.v3.XunitTestMethodRunnerContext",
+    MethodName = ".ctor",
+    ParameterTypeNames = ["_", "_", "_", "_", "_", "_", "_"],
```

Comment:
:stare:

Reply:
fixed in https://github.com/DataDog/dd-trace-dotnet/pull/6573/commits/0bb1ed8bf288df45f34491c03669193dd5f59b8b